### PR TITLE
Make sure the /local directory is created

### PR DESCRIPTION
**Pull request by betaredex**

### CODE REVIEW

File: openshift/pman-swift-publisher/get_data.py [hunk 1]
```diff
@@ -39,6 +39,8 @@ def getData(**kwargs):
     key = "input/data"
     success = True
 
+    if not os.path.exists('/local'):
```
Comment (danmcp) [inline openshift/pman-swift-publisher/get_data.py]: Why is one /local and the other not?

Comment (betaredex) [reply openshift/pman-swift-publisher/get_data.py]: oops

**Comment by danmcp** [comment]
@husky-parul Did this work for you?

**Comment by husky-parul** [comment]
@betaredex add `pip3 install python-keystoneclient` in Dockerfile. It is failing coz of missing dependency.

**Comment by husky-parul** [comment]
> @husky-parul Did this work for you?

It worked after adding pip3 install python-keystoneclient in Dockerfile

**Comment by betaredex** [comment]
Added it in setup.py since that's where the other dependencies are

**Comment by husky-parul** [comment]
> Added it in setup.py since that's where the other dependencies are

I think you need to add in /pman/openshift/pman-swift-publisher/Dockerfile as well
@danmcp correct me if I'm wrong.

**Comment by danmcp** [comment]
@husky-parul Sounds right

@betaredex Note that you have a merge conflict.  It looks like you pulled in some of the changes from the previous memory optimization PR.

**Pull request closed without merging**